vagrant/puppet/cmf/manifests/application/php.pp

```diff
@@ -0,0 +1,56 @@
+class cmf::application::php(
+  $version     = '5.5',
+  $use_apc     = undef,
+  $use_xdebug  = undef,
+  $environment = hiera('environment', 'dev')
+) {
+  validate_string($environment)
+
+  case $version {
+    '5.4':   { $ppaVersion = 'php5-oldstable' }
```

Comment:
@dbu @ElectricMaxxx do we need support for php 5.3?? The problem is that is it a bit difficult to install php-5.3 on a ubuntu trusty.